usaspending_api/etl/tests/integration/test_load_to_from_delta.py

```diff
@@ -177,24 +177,31 @@ def verify_delta_table_loaded_to_delta(
     else:
         expected_table_name = delta_table_name.split(".")[-1]
 
-    partition_col = TABLE_SPEC[delta_table_name].get("partition_column")
+    # Resolve Column(s) to sort on in order to compare results
+    partition_column = TABLE_SPEC[delta_table_name].get("partition_column")
+    if partition_column:
+        # Make sure sort_columns is in list format
```

Comment:
It's clear from reading the code that it's "making sure sort_columns is in list format". No need to make an inline comment saying what the code does. Perhaps you could improve this comment by changing it to explain why you're "making sure sort_columns is in list format".